test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -292,7 +292,7 @@ export class DisplayTestApp {
     }
 
     IModelApp.applicationLogoCard =
-      () => IModelApp.makeLogoCard({ iconSrc: "DTA.png", iconWidth: 100, heading: "Display Test App", notice: "For internal testing" });
+      () => IModelApp.makeLogoCard({ iconSrc: "../../public/DTA.png", iconWidth: 100, heading: "Display Test App", notice: "For internal testing" });
```

Comment:
idk if we need this change since we say in the vite-config the public dir is .static-assets, and it should pick it up correctly from there

Reply:
you're right, the app picked up the file after the change. Done